src/Test.UnitTests.BinaryParsers/PEBinary/PEBinaryTests.cs

```diff
@@ -70,6 +74,19 @@ public void PEBinary_PdbIsStripped()
             }
         }
 
+        [Fact]
+        public void PEBinary_LanguagueCode()
```

Comment:
>PEBinary_LanguagueCode [](http://example.com/codeflow?start=20&length=22)

Language is misspelled in this name.

This name doesn't describe what is being tested here.

Reply:
Language is misspelled in this name.  --- found I have several places, searched and fixed all.

updated the name.